tests/__tests__/sitka.test.ts

```diff
@@ -1,17 +1,97 @@
+import { Dispatch } from "redux"
+import { createSitkaAndStore } from "../sitka-test"
+import { createAppStore, Sitka } from "../../src/sitka"
+
+export class SitkaMock<T = {}> extends Sitka {
+  public registeredModules: T
+ }
+
 describe("Sitka", () => {
 
-  test(`setDispatch dispatch was called`, () => {
+  describe('setDispatch', () => {
+    // Setup
+    const { sitka, store } = createSitkaAndStore()
+    let wasMockDispatched: boolean
+    beforeEach(() => {
+      wasMockDispatched = false
+    })
+
+    test(`setDispatch dispatch was called`, () => {
+      const mockDispatch: Dispatch = <T>(action: T) => {
+        wasMockDispatched = true
+        store.dispatch(action)
+        return action
+      }
+
+      sitka.setDispatch(mockDispatch)
+      sitka.getModules().text.handleUpdateSize(20)
+
+      const { text: actual } = store.getState()
+      const expected = {
+        size: 20,
+        value: "Hello World",
+        numberOfEdits: 0,
+      }
+      expect(actual).toEqual(expected)
+      expect(wasMockDispatched).toEqual(true)
+    })
   })
 
-  test(`getModules returns Redux store`, () => {
+  test(`getModules returns registered modules`, () => {
+    const sitkaMock = new SitkaMock<string>()
+    const expected = 'sitka modules'
+    sitkaMock.registeredModules = expected
+    const actual = sitkaMock.getModules()
```

Comment:
no need to mock it out, just validate that the modules your registered are the ones you get back out, from real sitka instance